include/seqan3/io/sequence/sequence_file_format.hpp

```diff
@@ -31,19 +31,52 @@
 // DAMAGE.
 //
 // ============================================================================
-// Author: Joerg Winkler <j.winkler AT fu-berlin.de>
-// ============================================================================
+
+/*!\file seqan3/io/sequence/sequence_file_format.hpp
```

Comment:
please remove `seqan3/io/sequence/sequence_file_format.hpp` this information is not needed